### PR TITLE
Allow commas as a separator between object/interface type members.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**

### CODE REVIEW

**Comment by DanielRosenwasser** [comment]
:shipit:

**Comment by yuit** [comment]
:+1:

**Comment by mhegazy** [comment]
:+1:

File: tests/baselines/reference/ExportObjectLiteralAndObjectTypeLiteralWithAccessibleTypesInNestedMemberTypeAnnotations.types [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        constructor(public x: number, public y: number) { }
+>x : number
+>y : number
+    }
+
+    export var UnitSquare : {
+>UnitSquare : { top: { left: Point; right: Point; }; bottom: { left: Point; right: Point; }; }
+
+        top: { left: Point, right: Point },
+>top : { left: Point; right: Point; }
```
Comment (JsonFreeman) [inline tests/baselines/reference/ExportObjectLiteralAndObjectTypeLiteralWithAccessibleTypesInNestedMemberTypeAnnotations.types]: Should we print the type with a comma too?

**Comment by JsonFreeman** [comment]
:+1:

**Comment by mhegazy** [comment]
is this ready to go in or are we waiting on something?

**Comment by JsonFreeman** [comment]
We are in discussion as to whether the change should include members of interface declarations. Personally, I think it should, because interfaces are the same as object types.

**Pull request merged**